aws-xray-recorder-sdk-core/src/main/resources/META-INF/native-image/com.amazonaws/aws-xray-recorder-sdk-core/resource-config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "resources" : {
+    "includes" : [ {
+      "pattern" : "\\Qcom/amazonaws/xray/interceptors/DefaultOperationParameterWhitelist.json\\E"
```

Comment:
What do the `Q` and `E` mean? I assume just delimiters for beginning and ending the expression, just strange the usual regex characters aren't used.

Reply:
The pattern attribute is a Java regexp that matches resource(s) to be included in the image from https://www.graalvm.org/reference-manual/native-image/Resources/

I imagine this was created using the [tracing agent](https://www.graalvm.org/reference-manual/native-image/Agent/) so might have an auto generated feel :)